app/src/main/res/layout/activity_parking_stay.xml

```diff
@@ -0,0 +1,206 @@
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:ems="5"
+        android:fontFamily="sans-serif-black"
+        android:text="@string/end_time_selected_text_view"
+        android:textAlignment="center"
+        android:textColor="#FFFEFE"
+        app:layout_constraintBottom_toTopOf="@id/parking_lot_edit_text"
+        app:layout_constraintLeft_toRightOf="@id/end_date_text_view"
+        app:layout_constraintRight_toLeftOf="@id/end_date_time_button"
+        app:layout_constraintTop_toBottomOf="@id/end_date_selected_text_view" />
+
+    <TextView
+        android:id="@+id/parking_lot_text_view"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:fontFamily="sans-serif-black"
+        android:text="@string/parking_lot_text"
+        android:textAppearance="@style/TextAppearance.AppCompat.Body2"
+        android:textColor="#FFFFFF"
+        app:layout_constraintBottom_toTopOf="@id/security_code_text_view"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toLeftOf="@id/parking_lot_edit_text"
+        app:layout_constraintTop_toBottomOf="@id/end_date_text_view" />
+
+    <TextView
+        android:id="@+id/security_code_text_view"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:fontFamily="sans-serif-black"
+        android:text="@string/security_code_text"
+        android:textAppearance="@style/TextAppearance.AppCompat.Body2"
+        android:textColor="#FFFFFF"
+        app:layout_constraintBottom_toTopOf="@id/cancel_button"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toLeftOf="@id/security_code_edit_text"
+        app:layout_constraintTop_toBottomOf="@id/parking_lot_text_view" />
+
+    <EditText
+        android:id="@+id/parking_lot_edit_text"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:autofillHints=""
+        android:backgroundTint="#FFFFFF"
+        android:backgroundTintMode="src_over"
+        android:ems="8"
+        android:fontFamily="sans-serif-black"
+        android:inputType="number"
+        android:minHeight="48dp"
+        android:textAlignment="center"
+        android:textColor="#FFFFFF"
+        app:layout_constraintBottom_toBottomOf="@id/parking_lot_text_view"
+        app:layout_constraintLeft_toRightOf="@id/parking_lot_text_view"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toTopOf="@id/parking_lot_text_view"
+        tools:ignore="SpeakableTextPresentCheck" />
+
+    <EditText
+        android:id="@+id/security_code_edit_text"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:autofillHints=""
+        android:backgroundTint="#FFFFFF"
+        android:backgroundTintMode="src_over"
```

Comment:
whats the meaning of this property?

Reply:
Its shows to the user what he should write in the edit text. PE: a password, a name, age

Reply:
yes but I dont understand Why you are using this android:backgroundTintMode="src_over"